ray-operator/controllers/ray/rayjob_controller_unit_test.go

```diff
@@ -321,15 +321,15 @@ func TestUpdateRayJobStatus(t *testing.T) {
 
 func TestValidateRayJobSpec(t *testing.T) {
 	err := validateRayJobSpec(&rayv1.RayJob{})
-	assert.Error(t, err, "The RayJob is invalid because both `RayClusterSpec` and `ClusterSelector` are empty")
```

Comment:
Several subtests don't return the error from the validation rule we want to test, so I use `ErrorContains` instead.